wm_resources.rc

```diff
@@ -0,0 +1,88 @@
+#include "resource.h"
```

Comment:
To keep the code base and build as simple as possible, I think it would be better and simpler to put the default config as a text string in a header file and use it directly in the code

Reply:
Absolutely could be done but with keeping the default config in a separate file as it would appear when the application puts it in the directory, I think it would make it easier to modify as features get implemented instead of modifying a string in a header. Not to mention if an icon is ever added to this executable the resource file will have to come back, the resource file can also be used to implement translations for other languages for error messages in the future.

Reply:
That is indeed an upside, in that case, let's keep this resource file, but can you remove all the redundant comments and additional stuff and just leave the IDS_LIGHTWM_CNF_NAME? all the around is a lot of clutter in my eyes 😂

Reply:
Absolutely can clean it up.